fmriprep/workflows/base.py

```diff
@@ -424,13 +431,44 @@ def init_single_subject_wf(subject_id):
             getattr(fmap_wf.inputs, f"in_{estimator.bids_id}").metadata = [
                 s.metadata for s in estimator.sources
             ]
-            continue
 
-        if estimator.method == fm.EstimatorType.PEPOLAR:
+        elif estimator.method == fm.EstimatorType.PEPOLAR:
             raise NotImplementedError(
                 "Sophisticated PEPOLAR schemes are unsupported."
             )
-        # TODO: SyN fieldmap processing
+
+        elif estimator.method == fm.EstimatorType.ANAT:
+            from sdcflows.workflows.fit.syn import init_syn_preprocessing_wf
+            sources = [str(s.path) for s in estimator.sources if s.suffix == "bold"]
+            layout = config.execution.layout
+            syn_preprocessing_wf = init_syn_preprocessing_wf(
+                omp_nthreads=config.nipype.omp_nthreads,
+                debug=config.execution.sloppy,
+                auto_bold_nss=True,
+                t1w_inversion=True,
```

Comment:
an aside: i don't really understand this option - why are we inverting the t1w? cc @oesteban

Reply:
Because T2* contrast is generally inverted from T1w, we invert to give the registration algorithm a more similar target.

We haven't really tested keeping it static and using, e.g., a mutual information metric.